### PR TITLE
New header ty_header_60_cloud in if_aff_types_v1

### PR DESCRIPTION
**Pull request by huber-nicolas**
This header can be used for DMON, WMPC, CHKV, CHKO, CHKC, CHDO

### CODE REVIEW

**Comment by wurzka** [comment]
... and APLO

File: file-formats/zif_aff_types_v1.intf.abap [hunk 1]
```diff
@@ -6,25 +6,37 @@ INTERFACE zif_aff_types_v1 PUBLIC.
   "! The ABAP file format version
   TYPES ty_format_version TYPE string.
 
-  "! <p class="shorttext">ABAP Language Version</p>
+  "! <p class="shorttext">ABAP Language Version (Non-Source Code Objects)</p>
```
Comment (schneidermic0) [inline file-formats/zif_aff_types_v1.intf.abap:9]: We shouldn't add this to the title. I thought we remove it from the titles. If would be inherited by object types which reuse `ty_abap_language_version` 
```suggestion
  "! <p class="shorttext">ABAP Language Version</p>
```

If wanted, we can keep it in the description.

File: file-formats/zif_aff_types_v1.intf.abap [hunk 2]
```diff
@@ -6,25 +6,37 @@ INTERFACE zif_aff_types_v1 PUBLIC.
...
   "! ABAP language version
+  "! $values {@link zif_aff_types_v1.data:co_abap_language_version}
+  "! $default {@link zif_aff_types_v1.data:co_abap_language_version.standard}
   TYPES ty_abap_language_version TYPE c LENGTH 1.
 
+  "! <p class="shorttext">ABAP Language Version (Non-Source Code Objects)</p>
```
Comment (schneidermic0) [inline file-formats/zif_aff_types_v1.intf.abap:15]:
```suggestion
  "! <p class="shorttext">ABAP Language Version</p>
```

File: file-formats/zif_aff_types_v1.intf.abap [hunk 3]
```diff
@@ -6,25 +6,37 @@ INTERFACE zif_aff_types_v1 PUBLIC.
...
+  "! ABAP language version (without key user)
+  "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
+  "! $default {@link zif_aff_types_v1.data:co_abap_language_version_cloud.standard}
+  TYPES ty_abap_language_version_cloud TYPE c LENGTH 1.
+
+  "! <p class="shorttext">ABAP Language Version (Source Code Objects)</p>
```
Comment (schneidermic0) [inline file-formats/zif_aff_types_v1.intf.abap:21]:
```suggestion
  "! <p class="shorttext">ABAP Language Version</p>
```

File: file-formats/zif_aff_types_v1.intf.abap [hunk 4]
```diff
@@ -6,25 +6,37 @@ INTERFACE zif_aff_types_v1 PUBLIC.
...
+  "! ABAP language version for source code objects
+  "! $values {@link zif_aff_types_v1.data:co_abap_language_version_src}
+  "! $default {@link zif_aff_types_v1.data:co_abap_language_version_src.standard}
+  TYPES ty_abap_language_version_src TYPE c LENGTH 1.
+
   CONSTANTS:
     "! <p class="shorttext">ABAP Language Version (Source Code Objects)</p>
```
Comment (schneidermic0) [inline file-formats/zif_aff_types_v1.intf.abap:28]:
```suggestion
    "! <p class="shorttext">ABAP Language Version</p>
```

File: file-formats/zif_aff_types_v1.intf.abap [hunk 5]
```diff
@@ -73,17 +89,28 @@ INTERFACE zif_aff_types_v1 PUBLIC.
...
-      "! $values {@link zif_aff_types_v1.data:co_abap_language_version_src}
-      "! $default {@link zif_aff_types_v1.data:co_abap_language_version_src.standard}
-      abap_language_version TYPE ty_abap_language_version,
+      original_language     TYPE ty_original_language,
+      "! <p class="shorttext">ABAP Language Version</p>
+      "! ABAP language version
+      abap_language_version TYPE ty_abap_language_version_src,
     END OF ty_header_60_src.
 
+   TYPES:
```
Comment (schneidermic0) [inline file-formats/zif_aff_types_v1.intf.abap]: Please fix the indentation issue. The source code formatter might help, too ;)

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
I thinks we shouldn't add the information about source code objects and non-source code objects to the title of the types

**Comment by schneidermic0** [comment]
... and NROB see also #320

**Pull request merged**